gradle-plugin/README.md

```diff
@@ -1,207 +0,0 @@
-# Mozilla Application Services Gradle Plugin
-
-Plugin for consuming Mozilla Application Services megazord native libraries.
-
-<p align="left">
-    <a alt="Version badge" href="https://plugins.gradle.org/plugin/org.mozilla.appservices.gradle-plugin">
-        <img src="https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/org/mozilla/appservices/org.mozilla.appservices.gradle.plugin/maven-metadata.xml.svg?label=org.mozilla.appservices&colorB=brightgreen" /></a>
-</p>
-
-## Overview
-
-Mozilla Application Services publishes many native (Rust) code libraries that stand alone: each
-published Android ARchive (AAR) contains managed code (`classes.jar`) and multiple `.so` library
-files (one for each supported architecture).  That means consuming multiple such libraries entails
-at least two `.so` libraries, and each of those libraries includes the entire Rust standard library
-as well as (potentially many) duplicated dependencies.  To save space and allow cross-component
-native-code Link Time Optimization (LTO, i.e., inlining, dead code elimination, etc) Application
-Services also publishes composite libraries -- so called *megazord libraries* or just *megazords* --
-that compose multiple Rust components into a single optimized `.so` library file.  The managed code
-can be easily configured to use such a megazord without additional changes.
```

Comment:
There's some good "motivational" docs here about megazords that is going to go missing when we delete this plugin; should we move some of this into a new doc about the new megazord setup?